scaladoc-testcases/src/tests/enumSignatures.scala

```diff
@@ -9,19 +9,19 @@ enum Enum1
   case C
 }
 enum Enum2(val i: Int):
-  case A(val s: String) extends Enum2(1)
-  case B(val t: String) extends Enum2(2)
-  case C(val u: String) extends Enum2(3)
+  case A(s: String) extends Enum2/*<-*/(1)/*->*/
```

Comment:
What is the reason for these funny inlined comments?

Reply:
These arrows are marking fragments of signature that shouldn't be documented. We don't document constructors of parents and test would fail without that.